docs/advanced-config/README.md

```diff
@@ -92,6 +92,7 @@ Style strings are a list of words, separated by whitespace. The words are not ca
   - `bold`
   - `underline`
   - `dimmed`
+  - `reverse`
```

Comment:
I think it'd be good to have a short paragraph below describing what `reverse` does. It's not immediately obvious if this is color inversion or some sort of right-to-left token, but also the docs currently don't mention "inverted," which I think is what most people would be searching for.

I think after the paragraph on line 103 would be a good location (I'd put this comment there, but the GitHub interface won't let me)

Reply:
Actually, how about making an exception and using a different keyword than `ansi_term`? I agree that `reverse` is ambiguous. Using `inverted` is more explicit and self-documenting.

Reply:
Yeah, I wouldn't mind using `inverted`.

Reply:
Done. Should I still add a little blurb in the doc or do you think the keyword is sufficiently self-documenting?

Reply:
I think a single sentence in the previous paragraph would be sufficient now (e.g. ""inverted" swaps the background and foreground colors")